tests/encoding.rs

```diff
@@ -0,0 +1,31 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+mod common {
+    pub mod validation;
+}
+use common::validation::cr;
+
+use const_oid::db::rfc5912::SECP_256_R_1;
+use der::{Decode, Encode};
+use x509::request::CertReq; // for `app.oneshot()`
+
+#[test]
+fn reencode_multi() {
```

Comment:
Looks like tests in this file belong close to where this functionality is implemented, since these are isolated unit tests

Reply:
What do you mean? You want this in `lib.rs`?

Reply:
I'm not sure I understand the purpose of this test at all.
Isn't all of this functionality tested as part of bigger integration tests starting a real Steward with a real config and processing a real request?

Reply:
File deleted